crates/core/Cargo.toml

```diff
@@ -16,7 +16,8 @@ publish.workspace = true
 build = "build.rs"
 
 [lib]
-crate-type = ["staticlib", "rlib"]
```

Comment:
This isn't necessary, as the library name is already `liveview_native_core`, e.g. the static archive produced by rustc for a debug build is `target/debug/libliveview_native_core.a`, the same will be true of the other artifacts